reactor-netty-http/src/test/java/reactor/netty/http/client/Http2PoolTest.java

```diff
@@ -770,6 +856,79 @@ void maxLifeTime() throws Exception {
 		}
 	}
 
+	@Test
+	void evictionPredicate() throws Exception {
```

Comment:
Exception is never thrown in the method

```suggestion
	void evictionPredicate() {
```